content/client-lib-development-guide/features.textile

```diff
@@ -84,6 +84,11 @@ h3(#restclient). RestClient
 * @(RSC8)@ Supports two protocols:
 ** @(RSC8a)@ "MessagePack":http://msgpack.org/ binary protocol (this is the default for environments having a suitable level or support for binary data)
 ** @(RSC8b)@ JSON text protocol (used when @useBinaryProtocol@ option is false)
+** @(RSC8c)@ The library sets the @Accept@ and @Content-Type@ type headers, to reflect whether the client is configured to use a binary or JSON based protocol, to @application/x-msgpack@ or @application/json@ respectively.
```

Comment:
Unnecessary "type" plus, despite it being obvious to most  I would argue that it would help to insert the word "request" before headers to make it clear we're talking about the outbound communication from the client library to the server.

```suggestion
** @(RSC8c)@ The library sets the @Accept@ and @Content-Type@ request headers, to reflect whether the client is configured to use a binary or JSON based protocol, to @application/x-msgpack@ or @application/json@ respectively.
```

Reply:
fixed https://github.com/ably/docs/pull/888/commits/bfcb292156fecf01102086c64cd951f7b018c8b5